sauce.browsers.js

```diff
@@ -0,0 +1,141 @@
+const sauceBrowsers = {
```

Comment:
I checked just for fun, and turned out that `sauce.browsers` is more common name than `sauce-browsers` :-)
In `chai` they are using `sauce.browsers`, and no one using `sauce-browsers`.